### DISCUSSION TITLE
Mod Addition: Tiny Redstone

### DISCUSSION THREAD

**Discussion by iconmaster5326** [NillerMedDild/Enigmatica6#4445] [Ideas]
https://www.curseforge.com/minecraft/mc-mods/tiny-redstone

Tiny Redstone is honestly the best redstone mod I've been able to play with. It really helps with those bulky, ugly vanilla redstone setups. It'd be a great quality of life sort of mod.

**Comment by MuteTiefling** [comment]
The pack already contains a number of redstone 'enhancement' mods, like Immersive Engineering, Pneumaticcraft, rftools, redstone pens, Create, and more.
